src/citrine/resources/branch.py

```diff
@@ -165,30 +128,18 @@ def get(self,
             The requested version of the branch.
 
         """
-        if uid:
-            warnings.warn("Retrieving a branch by its version ID using .get() is deprecated. "
-                          "Please use .get_by_version_id().",
-                          DeprecationWarning)
-
-            if root_id:
-                raise ValueError("Please provide exactly one: the version ID or root ID.")
-            return self.get_by_version_id(version_id=uid)
-        elif root_id:
-            version = version or LATEST_VER
-            params = {"root": str(root_id), "version": version}
-            branch = next(self._list_with_params(per_page=1, **params), None)
-            if branch:
-                return branch
-            else:
-                raise NotFound.build(
-                    message=f"Branch root '{root_id}', version {version} not found",
-                    method="GET",
-                    path=self._get_path(),
-                    params=params
-                )
-
+        version = version or LATEST_VER
+        params = {"root": str(root_id), "version": version}
+        branch = next(self._list_with_params(per_page=1, **params), None)
+        if branch:
```

Comment:
```suggestion
        if branch is not None:
```
As long as we're editing